cluster/gce/container-linux/configure-helper.sh

```diff
@@ -1069,7 +1179,8 @@ function start-kube-controller-manager {
   # Calculate variables and assemble the command line.
   local params="${CONTROLLER_MANAGER_TEST_LOG_LEVEL:-"--v=2"} ${CONTROLLER_MANAGER_TEST_ARGS:-} ${CLOUD_CONFIG_OPT}"
   params+=" --use-service-account-credentials"
-  params+=" --cloud-provider=gce"
+  params+=" --cloud-provider=external"
+  params+=" --external-plugin=gce"
```

Comment:
What is this used for? Is this for some short-term shim?

Reply:
Essentially yes. VolumeController is not in the current round of extraction(https://goo.gl/CtzpVm). However VolumeController requires a working cloud. Unfortunately the relevant flag is conflating which cloud to use with which controllers to run. This is a slightly clumsy separation of which cloud provider to load from which plugins to run. When I did the original wiring I did not realize how deeply we had conflated these concepts. So right now I'm trying to make sure I can successfully break them. Once that is done I will come back and work out how to clean up the flags. Eventually we should be able to drop the need to specify and cloud provider. However I believe we will always need a way to specify which controllers to run.